angular/projects/admin-nrpti/src/app/import/import-csv/import-csv.component.ts

```diff
@@ -0,0 +1,299 @@
+import { Component } from '@angular/core';
+import { FactoryService } from '../../services/factory.service';
+import { ICsvTaskParams } from '../../services/task.service';
+import { CsvConstants, IRequiredFormat } from '../../utils/constants/csv-constants';
+import Papa from 'papaparse';
+import moment from 'moment';
+
+@Component({
+  selector: 'app-import-csv',
+  templateUrl: './import-csv.component.html',
+  styleUrls: ['./import-csv.component.scss']
+})
+export class ImportCSVComponent {
+  public dataSourceType = 'cors-csv'; // hard-code value while there is only 1 possible option
+  public recordType = 'Ticket'; // hard-code value while there is only 1 possible option
```

Comment:
Noticed the API as record type constants, while the front-end does not. Are there similar hardcoded constants in other areas of the front-end? Consistency between constants use across front-ends and api would be nice, but perhaps out of scope for this ticket.

Reply:
Yeah, its a bit all over the place at the moment.  I've tried to move more towards constants files, etc, but there are still lots of places with hard-coded values.

This particular instance is a bit weird, as this was written back when we were planning to have multiple types.  And so by default, there would be no pre-selected record or dataSource types, and the user would have to choose from some dropdowns. But since we have parred it back a bit and only have 1 option for each, it seemed silly to force the user to select the only options possible, so I've just auto-selected them by default.

Reply:
I see, and I suppose if we do need types again it's easy enough to switch out.